tools/update-tester/src/testrunner.rs

```diff
@@ -69,6 +98,42 @@ fn connect_to(config: &ConnectionConfig<'_>) -> TestClient {
     TestClient(client)
 }
 
+pub fn update_to_and_validate_new_toolkit_version(root_config: &ConnectionConfig) {
+    // update extension to new version
+    let test_db_name = "tsdb_toolkit_test";
+    let test_config = root_config.with_db(test_db_name);
+
+    let mut test_client = connect_to(&test_config);
+    test_client.update_to_current_version();
+
+    // run validation tests
+    let expected_results: QueryValues = vec![vec![
+        Some("100".to_string()),
+        Some("108".to_string()),
+        Some("149.5".to_string()),
+        Some("108.96220333142547".to_string()),
+        Some("109.50489521100047".to_string()),
+        Some("1.7995661075080858".to_string()),
+    ]];
+    test_client.validate_test_objects(expected_results);
```

Comment:
Sorry I didn't catch it before, @thatzopoulos, but we need these tests here too:

```
            test_client.check_no_references_to_the_old_binary_leaked(&current_version);
            test_client.validate_stable_objects_exist();
```